server/helpers/notification/notificateDelegateUtils.js

```diff
@@ -48,6 +49,64 @@ const sendEmailRewardCallNotificationToDelegates = async currentRoundInfo => {
   return subscribersToNotify
 }
 
+const sendEmailAfterBondingPeriodHasEndedNotificationToDelegates = async currentRoundInfo => {
```

Comment:
R: change name to: `sendEmailAfterBondingPeriodHasEndedNotificationToDelegators`

Reply:
Done